project/run.py

```diff
@@ -3,16 +3,25 @@
 import pytorch_lightning as pl
 from model import JeopardyModel
 from pytorch_lightning.loggers import WandbLogger
+import subprocess
 
-wandb_logger = WandbLogger(name='Adam-32-0.001',project='contra-pytorchlightning')
+PROJECT_NAME = "contra_pytorchlightning"
+wandb_logger = WandbLogger(name='from checkpoint with different hparams',project=PROJECT_NAME)
+
+CHECKPOINT_BASE_PATH = "/mnt/fs5/shashank2000/" + PROJECT_NAME + "/{}/checkpoints/epoch={}.ckpt" 
+class RealTimeEvalCallback(pl.Callback):
+    def on_validation_epoch_end(self, trainer, pl_module):
+        print(trainer.current_epoch)
+        if trainer.current_epoch % 2 == 0:
+            version = pl_module.logger.version
+            breakpoint()
```

Comment:
crap sorry this breakpoint was not supposed to stay in the commit, I'll take that off when I merge